### PR TITLE
Address sass division deprecation warnings when used in Dart Sass

### PR DESCRIPTION
**Pull request by bsteinbk**
Fix DEPRECATION WARNING when using with Dart Sass #131

### CODE REVIEW

File: src/user-icon/user-icon.scss [hunk 1]
```diff
@@ -81,8 +81,8 @@
       @include themed {
         fill: t(iui-icons-color);
       }
-      width: calc(100% / 1.5);
-      height: calc(100% / 1.5);
+      width: calc(100%* 0.6666667);
+      height: calc(100%* 0.6666667);
```
Comment (mayank99) [inline src/user-icon/user-icon.scss:85]: This really gives the same value? 😮

File: src/tooltip/tooltip.scss [hunk 2]
```diff
@@ -14,7 +14,7 @@
   width: max-content;
   overflow-wrap: break-word;
   box-sizing: content-box;
-  padding: round($iui-baseline / 3) $iui-s;
+  padding: round($iui-baseline * 0.3333) $iui-s;
```
Comment (mayank99) [inline src/tooltip/tooltip.scss:17]: And this

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,7 @@
 
 - **Improved wizard responsive design** by using percentage width.
 - **Fixed user-icon size not working when placed in icon button.**
+- **Fixed `DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.`.** when using scss files in project with Dart Sass 1.32.0 or later.
```
Comment (mayank99) [inline CHANGELOG.md:11]: Create a new section for this like so:

https://github.com/iTwin/iTwinUI-react/blob/272fd7fc33b1302d8aaa9a8c09bd4a5e1a623754/CONTRIBUTING.md#L153-L157

Also could reword it to focus on the :
"Fixed Sass deprecation warning about division operator** by replacing `/` with `*`.

**Review by mayank99 — COMMENTED** [review]
Thanks for the PR! We were thinking about how to tackle this warning (and `math.div` was not a good option since we use node-sass currently)

File: src/user-icon/user-icon.scss (hunk 1 above)
Comment (bsteinbk) [reply src/user-icon/user-icon.scss:85]: yep 1/1.5 = 0.6666667 - I messed up spacing I will fix that.

File: src/tooltip/tooltip.scss (hunk 2 above)
Comment (bsteinbk) [reply src/tooltip/tooltip.scss:17]: 1/3 = 0.33333

File: CHANGELOG.md (hunk 3 above)
Comment (bsteinbk) [reply CHANGELOG.md:11]: good point - will fix

File: src/user-icon/user-icon.scss (hunk 1 above)
Comment (mayank99) [reply src/user-icon/user-icon.scss:85]: Well, I'm more concerned about the precision

File: src/tooltip/tooltip.scss (hunk 2 above)
Comment (veekeys) [reply src/tooltip/tooltip.scss:17]: So is it 4 or 5 symbols after coma? :D

File: src/user-icon/user-icon.scss (hunk 1 above)
Comment (bsteinbk) [reply src/user-icon/user-icon.scss]: I have updated to use 0.66666666666666667 which produces the same results as 1/1.5 in test values I tried from 1 - 9600.

File: src/tooltip/tooltip.scss (hunk 2 above)
Comment (bsteinbk) [reply src/tooltip/tooltip.scss]: ditto here, I updated to 0.33333333333333333 to match values returned when dividing by 3

File: CHANGELOG.md (hunk 3 above)
Comment (bsteinbk) [reply CHANGELOG.md]: updated

File: src/tooltip/tooltip.scss (hunk 2 above)
Comment (veekeys) [reply src/tooltip/tooltip.scss]: oh...
@mayank99, could you quickly test from this branch generated css vs the current one if values produced are same?
Just to be sure that we need these "huge" numbers :D

Comment (mayank99) [reply src/tooltip/tooltip.scss]: So I checked the generated css, and then realized we are using `round` here so the precision would not matter after like 5 points.

And in that other example, we have `calc` so I think we don't even need to switch to `*` (because for [calc](https://developer.mozilla.org/en-US/docs/Web/CSS/calc()), `/` is a supported operator in native css).

File: src/user-icon/user-icon.scss [hunk 4]
```diff
@@ -67,8 +67,8 @@
     position: absolute;
     right: -2px;
     bottom: -2px;
-    width: calc(100% / 4);
-    height: calc(100% / 4);
+    width: calc(100% * 0.25);
```
Comment (bentleyvk) [inline src/user-icon/user-icon.scss:70]: Maybe just `25%`? 😀 Same with `66.66...%`

Comment (bsteinbk) [reply src/user-icon/user-icon.scss:70]: so there really is no need for calc.... just `width: 25%` right?

File: src/tooltip/tooltip.scss (hunk 2 above)
Comment (bsteinbk) [reply src/tooltip/tooltip.scss]: @mayank99 - so change it to 0.33?

File: src/user-icon/user-icon.scss (hunk 4 above)
Comment (mayank99) [reply src/user-icon/user-icon.scss:70]: Yeah. I would also go through all other instances of `calc` that you changed, to see if they can be simplified or undone (because using `/` in calc should not give you the deprecation warning).

File: src/tooltip/tooltip.scss (hunk 2 above)
Comment (mayank99) [reply src/tooltip/tooltip.scss]: Yup

File: src/user-icon/user-icon.scss (hunk 4 above)
Comment (mayank99) [reply src/user-icon/user-icon.scss:70]: Turns out "all other instances" are all in this file 😄

File: src/tooltip/tooltip.scss (hunk 2 above)
Comment (bsteinbk) [reply src/tooltip/tooltip.scss]: OK, I think I have it cleaned up now.

File: src/user-icon/user-icon.scss (hunk 4 above)
Comment (bsteinbk) [reply src/user-icon/user-icon.scss]: Updated to 25% and reverted others back to calc(100% / 1.5) since it was in calc function should be ignored by sass.

**Review by mayank99 — APPROVED** [review]
